deselected_tests.yaml

```diff
@@ -404,58 +302,10 @@ deselected_tests:
   - ensemble/tests/test_forest.py::test_importances[ExtraTreesRegressor-squared_error-float64]
   - cluster/tests/test_k_means.py::test_kmeans_elkan_results
 
-  # --------------------------------------------------------
-  # No need to test daal4py patching
 reduced_tests:
   - cluster/tests/test_affinity_propagation.py
-  - cluster/tests/test_bicluster.py
```

Comment:
@ethanglaser I don't know what exactly reduced_tests is used for (or if anything calls run_xpu_tests with 'reduced'). Will this section impact overall runtime (i.e. should we readd these to reduce CI times)? I'm leaning towards these staying in deselected_tests, but I need another opinion.

Reply:
@icfaust yes, I don't understand the difference between reduce_tests section and the ordinary deselected_tests section.

Reply:
It looks like we are running with reduced flag in internal CI. @homksei do you know what the reduced section is intended for?

Reply:
Honestly, I do not have information about the reasons why this was done.